projects/plugins/jetpack/class-jetpack-stats-dashboard-widget.php

```diff
@@ -89,69 +75,94 @@ public static function dashboard_widget() {
+					</p>
+					<p><?php echo esc_html_x( 'Blocked spam comments.', '{#} Spam comments blocked by Akismet -- number is on a prior line, text is a caption.', 'jetpack' ); ?></p>
+				<?php elseif ( current_user_can( 'activate_plugins' ) && ! is_wp_error( validate_plugin( 'akismet/akismet.php' ) ) ) : ?>
+					<a href="
+					<?php
+					echo esc_url(
+						wp_nonce_url(
+							add_query_arg(
+								array(
+									'action' => 'activate',
+									'plugin' => 'akismet/akismet.php',
+								),
+								admin_url( 'plugins.php' )
+							),
+							'activate-plugin_akismet/akismet.php'
+						)
+					);
+					?>
+								" class="button button-jetpack">
+						<?php esc_html_e( 'Activate Anti-spam', 'jetpack' ); ?>
+					</a>
+				<?php else : ?>
+					<p><a href="<?php echo esc_url( 'https://akismet.com/?utm_source=jetpack&utm_medium=link&utm_campaign=Jetpack%20Dashboard%20Widget%20Footer%20Link' ); ?>"><?php esc_html_e( 'Anti-spam can help to keep your blog safe from spam!', 'jetpack' ); ?></a></p>
+				<?php endif; ?>
+			</div>
 		</div>
-
-		<div class="akismet">
-			<h3><?php esc_html_e( 'Anti-spam', 'jetpack' ); ?></h3>
-			<?php if ( is_plugin_active( 'akismet/akismet.php' ) ) : ?>
-				<p class="blocked-count">
-					<?php echo esc_html( number_format_i18n( get_option( 'akismet_spam_count', 0 ) ) ); ?>
-				</p>
-				<p><?php echo esc_html_x( 'Blocked spam comments.', '{#} Spam comments blocked by Akismet -- number is on a prior line, text is a caption.', 'jetpack' ); ?></p>
-			<?php elseif ( current_user_can( 'activate_plugins' ) && ! is_wp_error( validate_plugin( 'akismet/akismet.php' ) ) ) : ?>
-				<a href="
+		<div class="footer-links">
+			<?php
+				$jetpack_logo = new Jetpack_Logo();
+				echo $jetpack_logo->get_jp_emblem( true );// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			?>
+			<span>
 				<?php
-				echo esc_url(
-					wp_nonce_url(
-						add_query_arg(
-							array(
-								'action' => 'activate',
-								'plugin' => 'akismet/akismet.php',
-							),
-							admin_url( 'plugins.php' )
-						),
-						'activate-plugin_akismet/akismet.php'
-					)
-				);
+				if ( current_user_can( 'jetpack_manage_modules' ) ) :
+					$i18n_headers = jetpack_get_module_i18n( 'stats' );
+					?>
+			<a href="<?php echo esc_url( admin_url( 'admin.php?page=jetpack#/settings?term=' . rawurlencode( $i18n_headers['name'] ) ) ); ?>"
+				>
+					<?php
+					esc_html_e( 'Configure stats', 'jetpack' );
+					?>
+			</a>
+					<?php
+					endif;
 				?>
-							" class="button button-jetpack">
-					<?php esc_html_e( 'Activate Anti-spam', 'jetpack' ); ?>
-				</a>
-			<?php else : ?>
-				<p><a href="<?php echo esc_url( 'https://akismet.com/?utm_source=jetpack&utm_medium=link&utm_campaign=Jetpack%20Dashboard%20Widget%20Footer%20Link' ); ?>"><?php esc_html_e( 'Anti-spam can help to keep your blog safe from spam!', 'jetpack' ); ?></a></p>
-			<?php endif; ?>
-		</div>
+			|
```

Comment:
The `|` is outside the `if` so will display even if a user can't `jetpack_manage_modules`.

Reply:
Ah, good catch. Thanks! Fixed in 9fbd77ab1a007e58bf12c14e062ab8955ede4346